packages/common/editors/EditorContainer.js

```diff
@@ -54,6 +55,20 @@ class EditorContainer extends React.Component {
     if (!this.changeCommitted && !this.changeCanceled) {
       this.commit({key: 'Enter'});
     }
+    window.removeEventListener('scroll', this.setContainerTransform);
+  }
+
+  setContainerTransform = () => {
+    if (this.containerRef) {
+      this.containerRef.style.transform = this.calculateTransform();
+    }
+  }
+
+  calculateTransform = () => {
+    const { column, left, scrollLeft, top, scrollTop } = this.props;
+    const editorLeft = isFrozen(column) ? left : left - scrollLeft;
```

Comment:
Should we adjust left position using `pageXOffset` as well?

Reply:
I don't think left and right is a concern.

All of the grid width will be changed due to the window size. if the user needs to scroll left to right, he / she must drag the scroll bar, and by doing this the cell lost focus and become a mask.